city/models.py

```diff
@@ -0,0 +1,29 @@
+from sqlalchemy import Column, DateTime, Float, ForeignKey, Integer, String
+from sqlalchemy.orm import relationship
+
+from database import Base
+
+
+class DBCity(Base):
+    __tablename__ = "city"
+
+    id = Column(Integer, primary_key=True, index=True)
+    name = Column(String(64), nullable=False)
+    additional_info = Column(String(255), nullable=False)
+
+    temperature = relationship(
+        "DBTemperature",
+        back_populates="city",
+        uselist=False
```

Comment:
The `uselist=False` in the `temperature` relationship implies a one-to-one relationship. If multiple temperature records per city are expected, consider setting `uselist=True` or removing this parameter.